Sources/SwiftkubeClient/Config/KubernetesClientConfig.swift

```diff
@@ -228,3 +228,51 @@ private extension AuthInfo {
 		return nil
 	}
 }
+
+public extension KubernetesClientConfig {
+	static func loadConfigFromPath(logger: Logger, path: String) throws -> KubernetesClientConfig? {
```

Comment:
You could change the function to look more "swifty" by moving the `path` parameter to first position and dropping the `fromPath` part:

```swift
func load(fromPath: String, logger: Logger) throws -> KubernetesClientConfig? { ... }
``` 

The call-site would look like this then:

```swift
KubernetesClientConfig().load(fromPath: "/some/path/", logger: logger)
```

Reply:
ah yes, that reads much better! I will implement 👍